Content.Shared/Verbs/Verb.cs

```diff
@@ -13,8 +13,9 @@ public enum VerbType
         Interaction = 1,
         Activation = 2,
         Alternative = 4,
-        Other = 8,
-        All = 1+2+4+8
+        Examine = 8,
+        Other = 16,
+        All = 1+2+4+8+16
```

Comment:
Honestly just put `0x1F` at this point.

Reply:
Yeah in #6473 I just moved to using bit shifts with `(1 << 5) - 1` for all.

Reply:
I've just used Interaction | Activation | Alternative etc before, but I agree on using the bitshift for sanity.

Reply:
obsolete after verb classes